### ISSUE TITLE
Cant Compile on Visual studio community 2017

### ISSUE THREAD

**Issue by Freedom911** [UnrealCourse/05_TestingGrounds#5]
I am not able to compile on visual community 2017.I get the following error
**Can't save a makefile that has invalid contents.  See UBTMakefile.IsValidMakefile()**

**Comment by sampattuzzi** [comment]
Are you downloading the repo and compiling?

**Comment by Freedom911** [comment]
Yes I am  downloading the repo and compiling on ue 4.18.1. Actually in 4.18.1 there are some changes necessary otherwise it wont generate project file or compile .Take a look 
[https://answers.unrealengine.com/questions/718575/error-when-updating-to-418-generating-visual-studi.html](url)

**Comment by barakooda** [comment]
Same here,
it does not compile on 4.16 or 4.15

**Comment by sampattuzzi** [comment]
Have a look at the how the previous sections where updated and you can do
the same for this sections
